### PR TITLE
BAC-2203 New Skeleton

### PR DESCRIPTION
**Pull request by nutoniko**
Added 
"optimax/api-doc-bundle"
"optimax/health-check"
"symfony/maker-bundle"

### CODE REVIEW

**Review by e-v-gordeev — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
         "ekino/newrelic-bundle": "^2.0",
+        "optimax/api-doc-bundle": "^1.0",
+        "optimax/health-check": "2.0.0",
+        "sensio/framework-extra-bundle": "^5.4",
         "sentry/sentry-symfony": "^2.3",
-        "symfony/monolog-bundle": "^3.3"
+        "symfony/console": "4.3.*",
+        "symfony/dotenv": "4.3.*",
+        "symfony/flex": "^1.1",
+        "symfony/maker-bundle": "^1.12",
```
Comment (cergey-obr) [inline composer.json]: I think maker-bundle should be in dev section

Comment (nutoniko) [reply composer.json]: maker-bundle moved to dev

**Review by ehunov — APPROVED** [review]

**Review by Nurenberg — APPROVED** [review]

**Review by cergey-obr — APPROVED** [review]

**Review by MicroHappy — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
         "ext-iconv": "*",
         "ext-json": "*",
-        "symfony/console": "*",
-        "symfony/dotenv": "*",
-        "symfony/flex": "^1.1",
-        "symfony/framework-bundle": "*",
-        "symfony/yaml": "*",
         "ekino/newrelic-bundle": "^2.0",
+        "optimax/api-doc-bundle": "^1.0",
+        "optimax/health-check": "2.0.0",
```
Comment (Linfuby) [inline composer.json]: Why not `^2.0`?

Comment (nutoniko) [reply composer.json]: fixed

**Review by rakeev — APPROVED** [review]

**Review by dzamyatin — APPROVED** [review]

**Review by Linfuby — APPROVED** [review]

**Pull request merged**